test/helpers/email_helpers.rb

```diff
@@ -5,4 +5,7 @@ def last_email_link
     link = /href="([^"]*)"/.match(body)
     link[1]
   end
+  def email_confirmation
```

Comment:
yes to it will fetch the frist user from database to create mail preview in development mode.